### PR TITLE
propertyapiDocumentation

### PR DESCRIPTION
**Pull request by Hammond1979**
This API Documentation is to create, post, get ,edit and delete properties.

### CODE REVIEW

File: src/routes/authenticationRoute.js [hunk 1]
```diff
@@ -112,4 +112,303 @@ authenticationRouter.post('/signup', validateUserSignup, signupAgent);
...
+ *     description: retrive all properties from database
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *         schema:
+ *           $ref: '#/definitions/Properties'
+ *     responses:
+ *       200:
+ *         description: property uploaded successfully
+ *     example: {
```
Comment (nosisky) [inline src/routes/authenticationRoute.js]: Is the example not supposed to be an array of objects?

**Review by nosisky — CHANGES_REQUESTED** [review]

File: src/routes/authenticationRoute.js [hunk 2]
```diff
@@ -112,4 +112,303 @@ authenticationRouter.post('/signup', validateUserSignup, signupAgent);
...
+   agentProperties
+   );
+
+/**
+ * @swagger
+ * /property/{userId}:
+ *   put:
+ *     tags:
+ *       - Property Details
+ *     description: retrive all properties from database
```
Comment (nosisky) [inline src/routes/authenticationRoute.js]: Work on your description, why is everything the same?

File: src/routes/authenticationRoute.js (hunk 1 above)
Comment (Hammond1979) [reply src/routes/authenticationRoute.js]: Array have being added too all examples

File: src/routes/authenticationRoute.js (hunk 2 above)
Comment (Hammond1979) [reply src/routes/authenticationRoute.js]: All descriptions have being worked on.

**Pull request merged**